### PR TITLE
Add HAL icon

### LINKED ISSUE #9365: Add HAL icon
### Brand Name

HAL

### Website

https://hal.science/

### Popularity Metric

HAL it's the french openscience plateforme

### Official Resources for Icon and Color

https://www.ccsd.cnrs.fr/boite-a-logos/

### Additional Comments

You can find a svg version here : https://jpswalsh.github.io/academicons/

### Contributing

No

### PR DESCRIPTION
**Pull request by PriyansuMaurya**
**Issue:** closes #9365

**Similarweb rank:**
  

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -14905,6 +14905,17 @@
             "title": "Zyte",
             "hex": "B02CCE",
             "source": "https://www.zyte.com"
+        },
+        {
+            "title": "Zyte",
```
Comment (PratyushJayachandran) [inline _data/simple-icons.json:14910]: Do not add another icon in this PR

**Review by PratyushJayachandran — CHANGES_REQUESTED** [review]
Missing the SVG itself

**Comment by PriyansuMaurya** [comment]
> Missing the SVG itself

Would you like me to upload an SVG?

**Comment by PratyushJayachandran** [comment]
> Would you like me to upload an SVG?

Yes. New icons need both SVG and JSON data.

**Comment by PriyansuMaurya** [comment]
> Yes. New icons need both SVG and JSON data.

where should I upload svg?

**Comment by PratyushJayachandran** [comment]
> > Would you like me to upload an SVG?
> 
> Yes. New icons need both SVG and JSON data.

`icons/hal.svg`
Check contributing.md for more details. Check any of the open PRs new icons with all the checks passing for reference.

**Comment by PriyansuMaurya** [comment]
I closing this one and opening a new one due to merge issues.

**Pull request closed without merging**